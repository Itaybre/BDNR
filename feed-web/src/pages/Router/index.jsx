import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

import path from 'enums/path';
import Feed from 'pages/Feed';
import Home from 'pages/Home';

const Router = () => (
  <BrowserRouter>
    <Switch>
      <Route component={Feed} path={path.Feed} />
      <Route component={Home} path={path.Home} />
    </Switch>
  </BrowserRouter>
);

export default Router;
