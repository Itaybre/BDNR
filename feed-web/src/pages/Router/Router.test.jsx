import React from 'react';
import { render, cleanup } from '@testing-library/react';

import Router from '.';

describe('<Router />', () => {
  afterEach(() => {
    cleanup();
  });

  it('should render without crashing', () => {
    const { asFragment } = render(<Router />);

    expect(asFragment()).toMatchSnapshot();
  });
});
