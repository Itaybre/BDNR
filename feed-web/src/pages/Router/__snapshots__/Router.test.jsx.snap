// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Router /> should render without crashing 1`] = `
<DocumentFragment>
  <h1>
    Home
  </h1>
</DocumentFragment>
`;
